build.gradle.kts

```diff
@@ -0,0 +1,32 @@
+plugins {
+    java
+    id("org.springframework.boot") version "2.7.9"
+    id("io.spring.dependency-management") version "1.1.0"
+}
+
+group = "com.example"
+version = "0.0.1-SNAPSHOT"
+java.sourceCompatibility = JavaVersion.VERSION_17
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation("org.springframework.boot:spring-boot-starter-data-jpa")
+    implementation("org.springframework.boot:spring-boot-starter-web")
+    implementation("org.springframework.boot:spring-boot-starter-web-services")
+    implementation("org.projectlombok:lombok:1.18.22")
+    implementation("org.mapstruct:mapstruct:1.4.2.Final")
+    implementation("org.mapstruct:mapstruct-processor:1.4.2.Final")
+    implementation("org.springframework:spring-jdbc:6.0.4")
+    implementation("org.springframework.boot:spring-boot-starter-jdbc")
+    testImplementation("org.springframework.boot:spring-boot-starter-test")
+    runtimeOnly("org.postgresql:postgresql")
+    annotationProcessor("org.projectlombok:lombok")
+    annotationProcessor("org.mapstruct:mapstruct-processor:1.4.2.Final")
```

Comment:
Can we extract version numbers in separate variables? For example you have 1.4.2.Final for mapstruct in 3 places. If we have it in a variable, it will be easier to update it in the future.